MANUAL.txt

```diff
@@ -440,6 +441,7 @@ header when requesting a document from a URL:
 [reStructuredText]: http://docutils.sourceforge.net/docs/ref/rst/introduction.html
 [S5]: http://meyerweb.com/eric/tools/s5/
 [Slidy]: http://www.w3.org/Talks/Tools/Slidy/
+[SILE]: https://sile-typesetter
```

Comment:
Hi I think you forgot the TLD ` .org` here

Reply:
Thank you, I fixed the commit with this error when I rebased just now.